css/styles.css

```diff
@@ -48,289 +57,115 @@ img {
-    letter-spacing: 0.02em;
-}
-.mobile-menu-list-link:first-child{
-    margin-bottom: 24px;
-}
-
-.mobile-menu-list-link:focus {
-    color: var(--IRIS, #4D5AE5);
-}
-
-.mobile-list-item-social-list {
-    display: flex;
-    gap: 40px;
-
-
 }
 
 /* >>>>>>>>> END OF HEADER <<<<<<<<<<<< */
 
 /* >>>>>>>>>> MAIN <<<<<<<<<<<<<<<<< */
 
-.main {
-
-}
-
 /* Section business solutions*/
 
-.section-business-solutions {
-    background-color: #2E2F42;
-    margin: 0 auto;
+.hero-section {
+    background-color: var(--navy-blue);
     padding: 72px 0;
-    background-image: linear-gradient(to bottom, rgba(46, 47, 66, 0.70), rgba(46, 47, 66, 0.70)), url(../images/hero-mob.jpeg);
+    background-image: linear-gradient(to bottom, var(--hero-shadow), var(--hero-shadow)), url(../images/hero-mob.jpeg);
     background-repeat: no-repeat;
     background-size: cover;
     background-position: center;
 }
 
-@media screen and (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
-    .section-business-solutions {
-    background-image: linear-gradient(to bottom, rgba(46, 47, 66, 0.70), rgba(46, 47, 66, 0.70)), url(../images/hero-mob-2x.jpeg);
+@media screen and (min-device-pixel-ratio: 2),
+(min-resolution: 192dpi),
+(min-resolution: 2dppx) {
+    .section-hero {
+        background-image: linear-gradient(to bottom, var(--hero-shadow), var(--hero-shadow)), url(../images/hero-mob-2x.jpeg);
     }
-    
 }
 
-.container-business-solutins {
-}
-
-.section-business-header {
-    color: var(--WHITE, #FFF);
+.hero-header {
+    margin: 0 auto;
+    margin-bottom: 72px;
+    width: min-content;
+    color: var(--white);
     font-size: 36px;
     font-weight: 700;
     line-height: 1.11;
     letter-spacing: 0.02em;
     text-transform: capitalize;
     text-align: center;
-    margin-bottom: 72px;
-    padding: 0 60px;
 }
 
-.section-business-btn {
+.hero-btn {
     display: block;
     margin: 0 auto;
-    background-color: #4D5AE5;
-    color: #FFF;
+    padding: 16px 32px;
+    min-width: 169px;
+    background-color: var(--iris);
+    color: var(--white);
     font-size: 16px;
     font-weight: 500;
     line-height: 1.5;
     letter-spacing: 0.04em;
     cursor: pointer;
-    padding: 16px 32px;
-    border-radius: 4px;
-    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.15);
     border: none;
-    min-width: 169px;
-    transition-property: background-color;
-    transition-duration: 250ms;
-    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
-
+    border-radius: 4px;
+    box-shadow: 0px 4px 4px 0px var(--shadow-large);
+    transition: var(--background-color-250ms);
 }
 
-.section-business-btn:hover,
-.section-business-btn:focus {
-    background-color: #404BBF;
+.hero-btn:hover,
+.hero-btn:focus {
+    background-color: var(--ocean);
 }
 
-/* end section business solutions */
-
 /* section values */
 
-.section-values {
-}
-
-.container-values {
-}
-
-.section-values-header {
-    
-}
-
 .visually-hidden {
```

Comment:
What is the purpose of this class (and element itself)?

Reply:
Class .visually-hidden hides the header of the value section as per design there is no header. However, for the purpose of maintaining semantics on the page and ensuring correct work of search engines header was added. In perfect case, in header, the information that highlights the content of the section, be stated.